Examples/Equatable/AutoEquatable.stencil

```diff
@@ -0,0 +1,39 @@
+// swiftlint:disable file_length
+
+// MARK: - AutoEquatable for classes, protocols, structs
+{% for type in types.implementing.AutoEquatable %}{% if not type.kind == "enum" %}
+// MARK: - {{ type.name }} AutoEquatable
+{% if not type.kind == "protocol" %}extension {{ type.name }}: Equatable {} {% endif %}
+{% if type.supertype.implements.Equatable or type.supertype.implements.AutoEquatable %} THIS WONT COMPILE, WE DONT SUPPORT INHERITANCE for AutoEquatable {% endif %}
```

Comment:
equality for class inheritance is usually more complicated, we could update this later to use type recognition and cast to type for `==` operator but it will require some testing